dashboard/dashboard_config.yml

```diff
@@ -11,4 +11,39 @@ environment:
   # Comment out if not using public pretty-URL
   query_pretty_url: 'true'
   # Cookie root domain, set is using OAuth, used to remove a user's cookie
-  cookie_root_domain: 'system.o6s.io'
\ No newline at end of file
+  cookie_root_domain: 'system.o6s.io'
+  # Public URL for your GitHub app
+  # see https://github.com/settings/apps/
+  # github_app_url: https://github.com/apps/o6s-io
+  # Public URL for your GitLab instance
+  # gitlab_url: https://gitlab.o6s.io/
+  # SealedSecrets public key
+  # public_key: |
```

Comment:
I'm less keen on this approach. I think using a K8s secret would work better and be easier to maintain.